### PR TITLE
removed DatabaseContext from ManagerLayer

### PR DESCRIPTION
**Pull request by Luke-Clements**
# Description

Removed DatabaseContext from AuthorizationManager(session manager).

## Type of change

chore

# How Has This Been Tested?

No tests yet. This is tested along with the controller.

# Notes

Not sure why it included an additional three files with "changes." There are no differences between these files and the originals. The only file I changed is AuthorizationManager

### CODE REVIEW

File: Backend/UnitTesting/ClaimServiceUT.cs [hunk 1]
```diff
@@ -1,96 +1,96 @@
...
+using DataAccessLayer.Models;
+using System;
+using System.Linq;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using ServiceLayer.Services;
+
+namespace UnitTesting
+{
+    [TestClass]
+    public class ClaimServiceUT
```
Comment (alfredovargasjr) [inline Backend/UnitTesting/ClaimServiceUT.cs:11]: Are we still using claims? If so, just resolve this as resolve

**Review by alfredovargasjr — CHANGES_REQUESTED** [review]

File: Backend/UnitTesting/ClaimServiceUT.cs (hunk 1 above)
Comment (Luke-Clements) [reply Backend/UnitTesting/ClaimServiceUT.cs:11]: I think we are, for preventing access to certain items.

File: Backend/UnitTesting/App.config [hunk 2]
```diff
@@ -1,25 +1,25 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (ChrisMeyer7088) [inline Backend/UnitTesting/App.config:1]: This file should not have changed

File: Backend/UnitTesting/app.config [hunk 3]
```diff
@@ -1,40 +1,25 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (ChrisMeyer7088) [inline Backend/UnitTesting/app.config:1]: Why do we have two app.config files in UnitTesting?

**Review by ChrisMeyer7088 — CHANGES_REQUESTED** [review]

File: Backend/UnitTesting/app.config (hunk 3 above)
Comment (Luke-Clements) [reply Backend/UnitTesting/app.config:1]: Probably because my computer is dumb.

File: Backend/UnitTesting/App.config (hunk 2 above)
Comment (Luke-Clements) [reply Backend/UnitTesting/App.config:1]: There is actually no difference here, I don't know why it is complaining...

File: Backend/UnitTesting/app.config (hunk 3 above)
Comment (Luke-Clements) [reply Backend/UnitTesting/app.config:1]: Actually master has two of these files as well, so I don't know what the issue is.

File: Backend/UnitTesting/App.config (hunk 2 above)
Comment (alfredovargasjr) [reply Backend/UnitTesting/App.config:1]: This is just VS being stupid

File: Backend/UnitTesting/app.config (hunk 3 above)
Comment (alfredovargasjr) [reply Backend/UnitTesting/app.config:1]: I think the different versions of VS running on people's computer does this. One is App.config and the other is app.config. I guess combine them?

**Pull request closed without merging**